uaclient/status.py

```diff
@@ -317,21 +317,22 @@ def __init__(
 """
 PROMPT_FIPS_PRE_ENABLE = (
     """\
-This will install the FIPS core packages.
+This will install the FIPS packages. The Livepatch service will be unavailable.
+Warning: This action can take a few minutes and cannot be undone.
 """
     + PROMPT_YES_NO
 )
 PROMPT_FIPS_UPDATES_PRE_ENABLE = (
     """\
-This will install the FIPS core packages and will include priority updates
-with security fixes.
+This will install the FIPS packages including security updates.
+Warning: this will take some time and cannot be undone.
```

Comment:
Is the difference in the warning intentional?

Fips says "... a few minutes ..."
Fips-updates says "... some time ..."